.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -30,3 +41,6 @@ Debugger Type (if applicable): [New debugger or legacy debugger?]
 **Diagnostics file**
 The diagnostics file contains information such as the Python version, environments, loaded assemblies, configuration options, etc. This file makes it easier for us to reproduce and fix bugs, so we would really appreciate it if you could provide it to us. This file can be generated by going to Tools->Options->Python->Diagnostics.
 You can attach the file to this issue or email it to us at ptvshelp@microsoft.com. If you email it, please add a link to this issue.
+```
```

Comment:
Logs may be large so it may be less intrusive to move that section down here. Also it really similar to diagnostics.